robottelo/constants.py

```diff
@@ -83,6 +84,34 @@
 GCE_NETWORK_DEFAULT = 'default'
 GCE_EXTERNAL_IP_DEFAULT = True
 
+# AzureRM specific constants
+AZURERM_VALID_REGIONS = {
+    'eastasia': 'East Asia', 'southeastasia': 'Southeast Asia',
```

Comment:
For those of us missing the context - why is that a dict?

And would it make sense to generate dict dynamically? Looks like there is very simple transformation to make:
```python
AZURERM_VALID_REGIONS = ['East Asia', 'UK South', 'West US 2', ...]
AZURERM_VALID_REGIONS = {key.lower().replace(' ', '') : key for key in AZURERM_VALID_REGIONS}
```

Reply:
updated the same..

Reply:
earlier I was thinking to keep the region as a part of dict because `wrapanapi and UI` needed in caps however CLI/API needed in lower case ...
But i think the lower case could handle at test case level hence keeping only list of caps regions now.
Thanks for your input. please revisit.

Reply:
Maybe `helpers` could have function to transform wrapanapi form to CLI form, and from CLI form to wrapanapi form. This can be introduced along with first test that actually needs it.

I think keeping simple list in constants is better choice.